### PR TITLE
WIP search: claim mode

### PR DESCRIPTION
**Pull request by Panos512**
- Adds claim mode on literature search results.
- An author can now claim papers that he wrote.

(needs inspirehep/inspirehep-search-js#23 )

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -20,7 +20,7 @@
 # granted to it by virtue of its status as an Intergovernmental Organization
 # or submit itself to any jurisdiction.
 
-web: gunicorn inspirehep.wsgi -c gunicorn.cfg
+# web: gunicorn inspirehep.wsgi -c gunicorn.cfg
```
Comment (Panos512) [inline Procfile]: WIP

File: inspirehep/config.py [hunk 2]
```diff
@@ -1021,9 +1021,9 @@ def _(x):
 
 # OAuthclient
 # ===========
-orcid.REMOTE_APP['params']['request_token_params'] = {'scope': '/orcid-profile/read-limited /activities/update /orcid-bio/update'}
+orcid.REMOTE_SANDBOX_APP['params']['request_token_params'] = {'scope': '/orcid-profile/read-limited /activities/update /orcid-bio/update'}
 OAUTHCLIENT_REMOTE_APPS = dict(
-    orcid=orcid.REMOTE_APP,
+    orcid=orcid.REMOTE_SANDBOX_APP,
```
Comment (Panos512) [inline inspirehep/config.py]: WIP

File: inspirehep/config.py [hunk 3]
```diff
@@ -1052,7 +1052,7 @@ def _(x):
 
 # Web services and APIs
 # =====================
-BEARD_API_URL = None  # e.g. "http://beard.inspirehep.net/api"
+BEARD_API_URL = "http://127.0.0.1:8000/api"  # e.g. "http://beard.inspirehep.net/api"
```
Comment (Panos512) [inline inspirehep/config.py]: WIP

File: inspirehep/modules/search/views.py [hunk 4]
```diff
@@ -73,6 +78,106 @@ def search():
                            **ctx)
 
 
+@blueprint.route('/search/claimedRecords', methods=['GET'])
+def get_claimed_records():
+    full_name = request.values.get('full_name')
+    orcid_id = request.values.get('orcid_id')
+
+    return jsonify({
+        'records': [1390114, 1298805]
```
Comment (Panos512) [inline inspirehep/modules/search/views.py]: WIP

File: inspirehep/modules/search/views.py [hunk 5]
```diff
@@ -73,6 +78,106 @@ def search():
...
+                record.commit()
+                return 'Record Claimed'
+
+    return 'error'
+
+
+@blueprint.route('/search/claimableRecords', methods=['GET'])
+def get_claimable_records():
+    full_name = request.values.get('full_name')
+    full_name = 'Gao, Yuanning'
```
Comment (Panos512) [inline inspirehep/modules/search/views.py]: WIP

File: inspirehep/modules/theme/static/scss/inspirehep.scss [hunk 6]
```diff
@@ -58,3 +58,53 @@ $fa-font-path: "/static/node_modules/font-awesome/fonts";
 @import "../node_modules/toastr/toastr";
 
 @import "../node_modules/impact-graphs/style";
+
+
```
Comment (Panos512) [inline inspirehep/modules/theme/static/scss/inspirehep.scss]: WIP (it will go on it's own css file)

File: inspirehep/modules/search/views.py [hunk 7]
```diff
@@ -73,6 +79,117 @@ def search():
...
+    record = Record(record)
+
+    possible_authors = []
+
+    for author in record['authors']:
+        for id in author['ids']:
+            if id['value'] == users_orcid:
+                author['curated_relation'] = True
+                record.commit()
+                return 'Record Claimed'
```
Comment (jmartinm) [inline inspirehep/modules/search/views.py]: If it's a return to the Angular app it would be better to return a json response.

**Comment by jacquerie** [comment]
Postponed, it will be resurrected if necessary.

**Pull request closed without merging**